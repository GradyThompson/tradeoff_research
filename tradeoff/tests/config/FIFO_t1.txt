tests/job_sets/FIFO_t1.txt
tests/results/FIFO_t1.txt
tradeoff.schedulers.FIFO,FIFO,10
1