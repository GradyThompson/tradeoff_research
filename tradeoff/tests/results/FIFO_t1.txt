cost: 161
3,1
2,1
1,1
8,1
5,1
6,1
7,1
4,1
9,1
10,1
11,4
12,3
13,3
14,4
15,4
16,3
17,3
18,0
19,0
20,0