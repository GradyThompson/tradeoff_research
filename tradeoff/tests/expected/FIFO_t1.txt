cost 112
1,1
2,1
3,1
4,1
5,1
6,1
7,1
8,1
9,1
10,1
11,4
12,3
13,3
14,4
15,4
16,3
17,3
18,0
19,0
20,0